docs/troubleshooting-faq/common-issues.md

```diff
@@ -85,12 +85,12 @@ Please refer to [this guide for creating a user for SSH connections](https://doc
 
 - First, create a user in AWS IAM and assign the following permissions to the user:
 
-<Screenshot url='https://cdn.appcircle.io/docs/assets/1.png' />
+<ExternalScreenshot url='https://cdn.appcircle.io/docs/assets/1.png' width='1968px' height='744px' />
```

Comment:
@furkanarabaci There is a huge space at the top and bottom of the image. That will be a problem.

![Screenshot from 2023-11-28 07-04-27](https://github.com/appcircleio/appcircle-docusaurus/assets/7355362/4ad11dcb-c78f-463d-bd64-588169c59d66)

The same image should be like below (or a minor margin can be accepted).

![Screenshot from 2023-11-28 07-03-23](https://github.com/appcircleio/appcircle-docusaurus/assets/7355362/616c5eb7-7100-4e62-9297-6bc454fce625)